samples/dockerbuilds/ARM/Dockerfile

```diff
@@ -0,0 +1,97 @@
+
+#########################################
+# Download and Configure OpenSSL
+
+RUN wget https://github.com/openssl/openssl/archive/refs/tags/OpenSSL_1_1_1f.tar.gz \
+&& tar -xvf ./OpenSSL_1_1_1f.tar.gz \
+&& cd openssl-OpenSSL_1_1_1f \
+&& ./Configure linux-armv4 --prefix=${TOOLCHAIN_SYSROOT}/bin/ --openssldir=${OPENSSL_ROOT_DIR} shared \
+&& make CC=${TOOLCHAIN_BIN}/${TOOLCHAIN_PLATFORM}-gcc AR=${TOOLCHAIN_BIN}/${TOOLCHAIN_PLATFORM}-ar \
+&& make install \
+&& cd ..
+
+#########################################
+# Build curl
+
+RUN wget http://curl.haxx.se/download/curl-7.60.0.tar.gz \
+&& tar -xvf curl-7.60.0.tar.gz \
+&& cd curl-7.60.0 \
+&& ./configure --with-sysroot=${TOOLCHAIN_SYSROOT} --prefix=${TOOLCHAIN_SYSROOT}/${TOOLCHAIN_PLATFORM}/libc --target=${TOOLCHAIN_PLATFORM} --with-ssl --with-zlib --host=${TOOLCHAIN_PLATFORM} CC=${TOOLCHAIN_BIN}/${TOOLCHAIN_PLATFORM}-gcc AR=${TOOLCHAIN_BIN}/${TOOLCHAIN_PLATFORM}-ar \
+&& make \
+&& make install \
+&& cd ..
+
+#########################################
+#Build uuid
+
+RUN wget https://mirrors.edge.kernel.org/pub/linux/utils/util-linux/v2.32/util-linux-2.32-rc2.tar.gz \
+&& tar -xvf util-linux-2.32-rc2.tar.gz \
+&& cd util-linux-2.32-rc2/ \
+&& ./configure --prefix=${TOOLCHAIN_PREFIX} --with-sysroot=${TOOLCHAIN_SYSROOT} --target=${TOOLCHAIN_NAME} --host=${TOOLCHAIN_NAME} --disable-all-programs  --disable-bash-completion --enable-libuuid \
+&& make \
+&& make install \
+&& cd ..
+
+#########################################
+# Build Azure C SDK
+
+RUN git clone --recursive https://github.com/azure/azure-iot-sdk-c.git
+WORKDIR azure-iot-sdk-c
+RUN mkdir cmake
+WORKDIR cmake
+
+# Create a toolchain file on the fly
+RUN echo "SET(CMAKE_SYSTEM_NAME Linux)     # this one is important" > toolchain.cmake
+RUN echo "SET(CMAKE_SYSTEM_VERSION 1)      # this one not so much" >> toolchain.cmake
+RUN echo "SET(CMAKE_SYSROOT ${TOOLCHAIN_SYSROOT}/${TOOLCHAIN_PLATFORM}/libc)" >> toolchain.cmake
+RUN echo "SET(CMAKE_C_COMPILER ${TOOLCHAIN_BIN}/${TOOLCHAIN_PLATFORM}-gcc)" >> toolchain.cmake
+RUN echo "SET(CMAKE_CXX_COMPILER ${TOOLCHAIN_BIN}/${TOOLCHAIN_PLATFORM}-g++)" >> toolchain.cmake
+RUN echo "SET(CMAKE_FIND_ROOT_PATH ${TOOLCHAIN_SYSROOT}/${TOOLCHAIN_PLATFORM})" >> toolchain.cmake
+RUN echo "SET(CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER)" >> toolchain.cmake
+RUN echo "SET(CMAKE_FIND_ROOT_PATH_MODE_LIBRARY ONLY)" >> toolchain.cmake
+RUN echo "SET(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)" >> toolchain.cmake
+RUN echo "SET(OPENSSL_ROOT_DIR ${OPENSSL_ROOT_DIR})" >> toolchain.cmake
+RUN echo "SET(OPENSSL_CRYPTO_LIBRARY ${OPENSSL_ROOT_DIR}/libcrypto.so.1.1)" >> toolchain.cmake
+RUN echo "SET(OPENSSL_SSL_LIBRARY ${OPENSSL_ROOT_DIR}/libssl.so.1.1)" >> toolchain.cmake
+RUN echo "SET(OPENSSL_INCLUDE_DIR ${OPENSSL_ROOT_DIR}/include)" >> toolchain.cmake
+RUN cmake -DCMAKE_TOOLCHAIN_FILE=toolchain.cmake -Duse_default_uuid=ON -Duse_condition=OFF ..
+RUN make
+
+WORKDIR ../..
+
+CMD ["/bin/bash"]
```

Comment:
please add a new line to line number #97

Reply:
Fixed. Thank you.